svgnative/include/svgnative/SVGDocument.h

```diff
@@ -174,6 +175,34 @@ class SVG_IMP_EXP SVGDocument
      */
     void Render(const char* id, const ColorMap& colorMap, float width, float height);
 
+    /**
+     * Retrieves the bounds of the SVG Document.
+     *
+     * The bounds are as tightly computed as possible. The rendering port's canvas
+     * is used to calculate the bounds so the user must make sure that a context has been
+     * set, otherwise an assertion will fire.
+     */
+    bool GetBoundingBox(Rect& bounds);
+
+    /**
+     * Retrieves the bounds of the subtree of an element with the given XML ID.
+     *
+     * The bounds are as tightly computed as possible. The rendering port's canvas
+     * is used to calculate the bounds so the user must make sure that a context has been
+     * set, otherwise an assertion will fire.
+     */
+    bool GetBoundingBox(const char* id, Rect& bounds);
+
+#ifdef DEBUG_API
+    /**
+     * TODO: Remove this.
```

Comment:
TODO ? Do we want to keep it? If it is not for the public, move it to SVGDocumentImpl.h?

Reply:
@dirkschulze removing  DEBUG_API content from this file